source/geod24/LocalRest.d

```diff
@@ -303,22 +303,54 @@ public void sleep (Duration timeout)
         dg = If non-null, this delegate will be called when the timer fires
         periodic = Speficies if the timer fires repeatedly or only once
 
+    Returns:
+        a Timer instance with the ability to stop the timer
```

Comment:
I think it's important to separate the reason leading to the immediate change from the whole system architecture.
What do  mean by this ?
1) This PR changes `setTimer`.
2) Before it didn't return anything, now it returns `void`.
3) The reason we are doing this is because we want the ability to stop a `Timer`.

However, the third point is not the whole story. If we *only* wanted the ability to stop a timer, we could return a `delegate`. But what happened instead is that we got new information, and changed the system accordingly.

To summarize this long explanation: Don't think about what change you are doing now, but how the system look as a whole after your change. Document your change in the commit, document the system in the code.

In this case:
```suggestion
        A `Timer` instance with the ability to control the timer
```

Reply:
I understand what you're saying.
I thought the function could expand. But I couldn't document it.